composer.json

```diff
@@ -18,7 +18,8 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~1.8",
         "phpunit/phpunit": "~4.5",
-        "satooshi/php-coveralls": "~0.6"
+        "satooshi/php-coveralls": "~0.6",
+        "symfony/event-dispatcher": "~2.8|~3.0"
```

Comment:
Yes, but we have to ensure the correct version of the interface we are using. The library might be incompatible with `EventDispatcherInterface` version 4.0.